JSDemos/Demos/MultiView/Overview/description.md

```diff
@@ -1 +1,6 @@
-The MultiView component contains several views and allows an&nbsp;end user to&nbsp;switch between them. In&nbsp;this demo, all views are defined using a&nbsp;custom template. You can navigate through the views by&nbsp;swiping them in&nbsp;the horizontal direction.
\ No newline at end of file
+The MultiView component contains several views and allows users to switch between them. In this demo, to navigate through the views, swipe them in the horizontal direction.
+
+You can create view items in the [items](/Documentation/ApiReference/UI_Components/dxMultiView/Configuration/items/) array, or populate view items from a [dataSource](/Documentation/ApiReference/UI_Components/dxMultiView/Configuration/#dataSource). If you specify the [dataSource](/Documentation/ApiReference/UI_Components/dxMultiView/Configuration/#dataSource) structure as described in the [items](/Documentation/ApiReference/UI_Components/dxMultiView/Configuration/items/) API section, the [dataSource](/Documentation/ApiReference/UI_Components/dxMultiView/Configuration/#dataSource) populates the view items. Otherwise, you need to specify an [itemTemplate](/Documentation/ApiReference/UI_Components/dxMultiView/Configuration/#itemTemplate). This demo shows the implementation of the latter.
```

Comment:
I see nothing fundamentally wrong with "view items"... However, the first sentence lists two options:
1. An ability to create view items from scratch.
2. An ability to load view items from a data source. As for the word "populate" regarding view items themselves, I'm not that sure. You can populate something with view items.

```suggestion
You can create view items in the [items](/Documentation/ApiReference/UI_Components/dxMultiView/Configuration/items/) array, or load them from a [dataSource](/Documentation/ApiReference/UI_Components/dxMultiView/Configuration/#dataSource). If you specify the [dataSource](/Documentation/ApiReference/UI_Components/dxMultiView/Configuration/#dataSource) structure as described in the [items](/Documentation/ApiReference/UI_Components/dxMultiView/Configuration/items/) API section, the [dataSource](/Documentation/ApiReference/UI_Components/dxMultiView/Configuration/#dataSource) populates the component with view items. Otherwise, you need to specify an [itemTemplate](/Documentation/ApiReference/UI_Components/dxMultiView/Configuration/#itemTemplate). This demo shows how to implement the latter scenario.
```